awscli/customizations/cloudtrail/validation.py

```diff
@@ -486,8 +488,10 @@ def _load_and_validate_digest(self, public_keys, bucket, key):
         fingerprint = digest_data['digestPublicKeyFingerprint']
         if fingerprint not in public_keys:
             raise DigestError(
-                ('Digest file\ts3://%s/%s\tINVALID: public key not found for '
-                 'fingerprint %s') % (bucket, key, fingerprint))
+                ('Digest file\ts3://%s/%s\tINVALID: public key not found in %s '
```

Comment:
Maybe make it ``"in region %s"`` so they know that you are referring to a region?